docs/releasing.md

```diff
@@ -20,18 +20,30 @@ The highest type before a new release determines the version update number, so i
 
 ### Release Guide
 
-1. Create a branch `release-$version` in <https://github.com/owncloud/web>.
-2. Create a folder in `changelog` for the release version and date `mkdir $major.$minor.$patchVersion_YYYY-MM-DD`.
-3. Move all changelog items from the `changelog/unreleased/` folder to the `$major.$minor.$patchVersion_YYYY-MM-DD` folder.
-4. Run `./dev/scripts/bump_versions.sh $version`. This script will bump the `package.json` files in all relevant packages.
-5. Commit and push your changes.
-6. Run `./dev/scripts/create_and_push_tags.sh`. This script will create and push tags for the main app as well as all packages that need to be released.
-7. After merging, wait for the CI to run on the merge commit.
-8. Go to the [Releases section](https://github.com/owncloud/web/releases) and click "Draft a new Release".
-9. The release artifacts will be created automatically.
-
-### Next steps
-
-For oCIS the release assets need to be updated.
-
-<!-- TODO: add reference to docs on how to update Web in oCIS -->
+1. Checkout a stable branch
+   * If you are releasing a new major version, create a new branch `stable-$major` and push it
+   * If you are releasing a new minor or patch version, checkout the latest `stable-$major`
+   * If you are releasing a backport of a bugfix, checkout latest `stable-$major.$minor.$patch` (or `stable-$major` if patch branch is not available), create a new branch `stable-$major.$minor.$patch` and push it
```

Comment:
Reading this, I am not fully clear about it. Just some notes:
To be added  in the line after item 1 but before the bullet item: Replace the placeholders `$major`, `$minor` or `$patch` accordingly
minor/patch: ...checkout the latest `stable-$major` --> is this really `stable-$major` and if yes we should note it why. Need to add `and push it`
Backport also needs some love to make it more clear